src/rnn_api.cpp

```diff
@@ -36,6 +36,62 @@ enum RNNDir_t
     BackwardData,
     BackwardWeights
 };
+
+extern "C" miopenStatus_t
+miopenSetRNNDataSeqTensorDescriptor(miopenSeqTensorDescriptor_t seqTensorDesc,
+                                    miopenDataType_t dataType,
+                                    miopenRNNBaseLayout_t layout,
+                                    int maxSequenceLen,
+                                    int batchSize,
+                                    int vectorSize,
+                                    const int* sequenceLenArray,
+                                    void* paddingMarker)
+{
+    (void)paddingMarker;
```

Comment:
If we support C++17, shouldn't we use attribute `[[maybe_unused]]` instead of C-style cast to `void`?
```suggestion
                                    [[maybe_unused]] void* paddingMarker)
{
```